### PR TITLE
Updated Open File Limit in Administration Doc

### PR DESCRIPTION
**Pull request by asvetlik**
## Overview

Fixed the commit history from Pull Request #2043 

## Pull request checklist

- [x] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [x] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [x] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [x] I have resolved any merge conflicts.
- [x] I have included tests that cover my changes.
- [x] All new and existing tests pass.
- [ ] Make sure PR title conforms to convention in CHANGELOG.md.
- [ ] Add appropriate changelog label to PR (if applicable).

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.
- [ ] Make sure PR title conforms to convention in CHANGELOG.md.
- [ ] Make sure PR is tagged with appropriate changelog label.

### CODE REVIEW

File: docs/administration.md [hunk 1]
```diff
@@ -42,7 +42,26 @@ While Pilosa does have some high system requirements it is not a best practice t
...
 
-On Mac OS X, `ulimit` does not behave predictably. [This blog post](https://blog.dekstroza.io/ulimit-shenanigans-on-osx-el-capitan/) contains information about setting open file limits in OS X.
+On Mac OS X, `ulimit` does not behave predictably. The Mac OS X system has a utility called csrutil that prevents you from changing the file limit easily. Because of this, you must disable the csrutil program before attempting to set the open file limit. To disable the csrutil program, restart your laptop and when the start up screen pops up, hold down command + R to enter Reboot Mode. Open a terminal and enter `csrutil disable`, then restart your computer as you normally would. Now that the csrutil is disabled, you can change the open file limit. The open file limit can be changed by creating the following files and changing their ownership:
```
Comment (alanbernstein) [inline docs/administration.md]: "recovery mode", not "reboot mode"

Comment (alanbernstein) [inline docs/administration.md]: After the second sentence, I'd suggest a mild disclaimer, something like "One workaround that may work for you involves disabling this utility."

File: docs/administration.md [hunk 2]
```diff
@@ -42,7 +42,26 @@ While Pilosa does have some high system requirements it is not a best practice t
...
+
+```
+sudo vim /Library/LaunchDaemons/limit.maxfiles.plist
+```
+
+Copy the code located in [this](https://github.com/wilsonmar/mac-setup/blob/master/configs/limit.maxfiles.plist) github repository into the file.
```
Comment (alanbernstein) [inline docs/administration.md]: I'd suggest a slight rearrangement here:

Copy the contents of [this] file into a new file on your system located at `/Library/LaunchDaemons/limit.maxfiles.plist`, then run:

```sudo chown root:wheel /Library/LaunchDaemons/limit.maxfiles.plist```

File: docs/administration.md [hunk 3]
```diff
@@ -42,7 +42,26 @@ While Pilosa does have some high system requirements it is not a best practice t
...
+
+```
+sudo chown root:wheel /Library/LaunchDaemons/limit.maxfiles.plist
+sudo vim /Library/LaunchDaemons/limit.maxproc.plist
```
Comment (alanbernstein) [inline docs/administration.md]: And similar to the above comment for the maxproc step.

File: docs/administration.md [hunk 4]
```diff
@@ -42,7 +42,21 @@ While Pilosa does have some high system requirements it is not a best practice t
...
 
-On Mac OS X, `ulimit` does not behave predictably. [This blog post](https://blog.dekstroza.io/ulimit-shenanigans-on-osx-el-capitan/) contains information about setting open file limits in OS X.
+On Mac OS X, `ulimit` does not behave predictably. The Mac OS X system has a utility called csrutil that prevents you from changing the open file limit easily. One workaround that may work for you involves disabling the csrutil program. To disable the csrutil program, restart your laptop and when the start up screen pops up, hold down command + R to enter Recovery Mode. Open a terminal and enter `csrutil disable`, then restart your computer as you normally would. Now that the csrutil is disabled, you can change the open file limit. The open file limit can be changed by creating the following files and changing their ownership:
+
+Copy the contents of [this] file into a new file on your system located at /Library/LaunchDaemons/limit.maxfiles.plist, then run:
```
Comment (alanbernstein) [inline docs/administration.md]: Sorry, I was lazy and left out the url part of the link - we need to keep both of those.

Comment (asvetlik) [reply docs/administration.md]: It's all good. I should've caught that.

**Pull request merged**